app/views/static_pages/home.html.erb

```diff
@@ -0,0 +1,8 @@
+<% provide :title, "Home" %>
+
+<h1>Welcome to the Sample App</h1>
+<p> 
+This is the home page for the
+<a href="http://www.railstutorial.org/">Ruby on Rails Tutorial</a>
+sample application.
```

Comment:
indenting 2 spaces and using link_to instead of a tag

Reply:
done

Reply:
do you fix it yet?

Reply:
I will do it now. Just now I updated it in wrong file, sorry.